src/test/HostActivationTests/NativeHosting/Nethost.cs

```diff
@@ -142,6 +137,57 @@ public void GetHostFxrPath_HostFxrAlreadyLoaded()
                 .And.HaveStdErrContaining($"Found previously loaded library {HostFxrName}");
         }
 
+        [Theory]
+        [InlineData("{0}", true)]
+        [InlineData("{0}\n", true)]
+        [InlineData("{0}\nSome other text", true)]
```

Comment:
It is not clear why we should allow this case to be supported.

Reply:
This is for extensibility. If in the future we needed to add more stuff into the file, the "old" apphosts should not fail on it. So we intentionally only read the first line and ignore the rest.